datacatalog/google/cloud/datacatalog_v1beta1/gapic/data_catalog_client.py

```diff
@@ -255,8 +265,8 @@ def search_catalog(
         self,
         scope,
         query,
+        order_by,
```

Comment:
Hmm... Another breaking change. I figured https://github.com/googleapis/googleapis/commit/906335b5a958557e1070fc6f275171dc8a42b1ec#diff-8aa4e10bdf6632d55d2205632399043bL66 would have fixed that.

Reply:
Might be because it's not labeled as OPTIONAL in the request message. https://github.com/googleapis/googleapis/blob/906335b5a958557e1070fc6f275171dc8a42b1ec/google/cloud/datacatalog/v1beta1/datacatalog.proto#L468 I'll try this again.